bedrock/mozorg/templatetags/misc.py

```diff
@@ -223,6 +223,111 @@ def high_res_img(ctx, url, optional_attributes=None):
     return Markup(markup)
 
 
+@library.global_function
+@jinja2.pass_context
+def resp_img(ctx, url, srcset, sizes=None, optional_attributes=None):
+    alt = ""
+    attrs = ""
+    final_sizes = ""
+    final_srcset = ""
+    l10n = False
+
+    if optional_attributes:
+        l10n = optional_attributes.pop("l10n", False)
+        alt = optional_attributes.pop("alt", "")
+
+        if optional_attributes:
+            attrs = " " + " ".join(f'{attr}="{val}"' for attr, val in optional_attributes.items())
+
+    # default src
+    url = l10n_img(ctx, url) if l10n else static(url)
+
+    # srcset
+    srcset_last_item = list(srcset)[-1]
+    for image, size in srcset.items():
+        postfix = "" if image == srcset_last_item else ","
+        img = l10n_img(ctx, image) if l10n else static(image)
+        final_srcset = final_srcset + img + " " + size + postfix
+
```

Comment:
I'll give this some thought,  but I do wonder if the suggested code makes things harder to follow (whilst the bug it tries to prevent seems pretty low probability?)

Reply:
Yeah, i think it's v low risk as-is, so let's leave it. 

It's shame we can't (I think) easily use the `loop.last` behaviour that is available in a jinja loop (and there's a similar one in Django - `forloop.last`)

Reply:
Yeah, I was kinda surprised how tricky this was to do in Python. Seems to work well enough at least for now 🙂

Reply:
An alternative is to use `join` to handle the comma logic and also iteration...

```
if srcset:
    final_srcset = ",".join(
        [f"{l10n_img(ctx, image) if l10n else static(image)} {size}" for image, size in srcset.items()]
    )
```